hw/chisel_acc/src/main/scala/snax_acc/reshuffle/Reshuffle.scala

```diff
@@ -6,8 +6,8 @@ import chisel3.util._
 class ReshufflerCtrl(params: ReshufflerParams) extends Bundle {
 
   // packed CRS configuration
-  val reduceLen_i = UInt(5.W)
-  val tloop2Len_i = UInt(25.W)
+  val reduceLen_i = UInt(6.W)
+  val tloop2Len_i = UInt(24.W)
```

Comment:
This are really fix numbers or can they be parametrized? haha

Reply:
Can be both!